src/octokit_utils/index.ts

```diff
@@ -1,127 +1,188 @@
   try {
-    const res = await context.github.pulls.createReview(req)
+    const res = await context.github.pulls.createReview(req);
     if (res.status === 200) {
-      context.log.info('Approve changes succeeded.')
+      context.log.info("Approve changes succeeded.");
     } else {
-      context.log.error(`Approve change rejected with: ${JSON.stringify(res.data)}`)
+      context.log.error(
+        `Approve change rejected with: ${JSON.stringify(res.data)}`,
+      );
     }
   } catch (err) {
-    context.log.error(`Approve change failed with: ${JSON.stringify(err)}`)
+    context.log.error(`Approve change failed with: ${JSON.stringify(err)}`);
   }
-}
+};
 
-async function createPassingStatus(context: Context<Webhooks.WebhookPayloadPullRequest>, startTime: string, endTime: string): Promise<void> {
-  const statusOptions: (Octokit.RequestOptions & Octokit.ChecksCreateParams) = context.repo({
-    name: APP_CHECK_NAME,
-    head_sha: context.payload.pull_request.head.sha,
-    status: 'completed',
-    started_at: startTime,
-    completed_at: endTime,
-    conclusion: 'success',
-    output: {
-      title: 'test',
-      summary: 'test',
-      text: 'test',
-    },
-    request: {
-      retries: 3,
-      retryAfter: 3,
-    },
-  })
+const createPassingStatus = async (
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+  startTime: string,
+  endTime: string,
+): Promise<void> => {
+  const statusOptions: Octokit.RequestOptions &
+    Octokit.ChecksCreateParams = context.repo({
+      completed_at: endTime,
+      conclusion: "success",
+      head_sha: context.payload.pull_request.head.sha,
+      name: APP_CHECK_NAME,
+      output: {
+        summary: "test",
+        text: "test",
+        title: "test",
+      },
+      request: {
+        retries: 3,
+        retryAfter: 3,
+      },
+      started_at: startTime,
+      status: "completed",
+    });
   const response = await context.github.checks.create(statusOptions);
-  context.log.info(`Create passing status finished with status ${response.status}`);
+  context.log.info(
+    `Create passing status finished with status ${response.status}`,
+  );
   if (response.status !== 201) {
-    context.log.error(`Create passing status failed with status ${response.status} and error: ${JSON.stringify(response.data)}`)
+    context.log.error(
+      `Create passing status failed with status ${
+        response.status
+      } and error: ${JSON.stringify(response.data)}`,
+    );
   }
-}
+};
 
-async function getChangedFiles(context: Context<Webhooks.WebhookPayloadPullRequest>) {
-  const changedFilesResponse = await context.github.pulls.listFiles(initPullRelatedRequest(context))
-  const changedFiles: string[] = []
+const getChangedFiles = async (
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+): Promise<string[]> => {
+  const req = initPullRelatedRequest(context) as unknown as (Octokit.RequestOptions & Octokit.PullsListFilesParamsDeprecatedNumber);
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)